### PR TITLE
Release notes 1.6.3

### PR DESCRIPTION
**Pull request by stevshap**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -3,6 +3,13 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## 1.6.3
+### Enhancements:
+- Verrazzano analysis tooling enhancements.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:8]: Can we put any specific information here, like Now you can analyze clusters for issues with x,y,z.

**Review by rosemarymarano — COMMENTED** [review]
Minor improvement suggested; would like to see the identical 1.6.2 enhancement improved also.

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -3,6 +3,13 @@ title: Release Notes
 weight: 2
 draft: false
 ---
+## 1.6.3
+### Enhancements:
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:7]: Just was advised that the subordinate headings under 1.6.3 and 1.6.2 are head 3's and should be head 4's. Would you please fix them in both sections?

**Review by rosemarymarano — COMMENTED** [review]
One more minor fix needed.

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (stevshap) [reply content/en/docs/releasenotes/_index.md:7]: ok so it should be
## 1.6.3
#### enhancements...
?

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (stevshap) [reply content/en/docs/releasenotes/_index.md:8]: Still no update from developers on specifics...

File: content/en/docs/releasenotes/_index.md (hunk 2 above)
Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:7]: Yes.

File: content/en/docs/releasenotes/_index.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/releasenotes/_index.md:8]: I know. I as checking past RNs and on a previous release, Michael Gian updated that enhancement statement to "Verrazzano CLI tool validates that commands are entered correctly." (Of course, that doesn't help us now.) I think that some "enhancements" may have been the ability to report CAPI and Rancher issues (Michael Gian and Bob Donat)

**Review by rosemarymarano — COMMENTED** [review]
Few minor edits.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**